bukkit/src/main/java/me/william278/husksync/bukkit/config/ConfigLoader.java

```diff
@@ -24,6 +24,9 @@ public static void loadSettings(FileConfiguration config) throws IllegalArgument
         Settings.syncAdvancements = config.getBoolean("synchronisation_settings.advancements", true);
         Settings.syncLocation = config.getBoolean("synchronisation_settings.location", false);
         Settings.syncFlight = config.getBoolean("synchronisation_settings.flight", false);
+
+        // Future
+        Settings.useNativeImplementation = config.getBoolean("useNativeImplementation", false);
```

Comment:
Use snake_case for config strings (use_native_implementation). Possibly also rename this option; `native_advancement_synchronization` or something?